src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -211,6 +211,42 @@
   <data name="SplitPaneText" xml:space="preserve">
     <value>Split pane</value>
   </data>
+  <data name="SplitPaneDownText" xml:space="preserve">
+    <value>Split Pane Down</value>

```

Comment:
nit: I can't remember, but I _believe_ our current style isn't to capitalize the first letter of each word for commands, only the first word.